src/main/java/com/streamforge/service/common/security/EncryptService.java

```diff
@@ -0,0 +1,39 @@
+package com.streamforge.service.common.security;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+
+@Service
+public class EncryptService {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(EncryptService.class);
+
+    public String encrypt(String toEncrypt) {
+        try {
+            MessageDigest hasher = MessageDigest.getInstance("SHA-256");
+            byte[] digest = hasher.digest(toEncrypt.getBytes(StandardCharsets.UTF_8));
+            return bytesToHex(digest);
```

Comment:
Hmm, what do you mean?

Reply:
return "" is not quite informative, i can be wrong, It would be much informative if we gonna have 
String encryptedData = "";
try
encrypted data = hash(...)
catch

return encryptedData;

Reply:
Thanks, however, the comment was not really about the exception